CHANGELOG.md

```diff
@@ -77,7 +77,7 @@ If you're using HTML or custom JavaScript, change:
 
 #### If you’re using Nunjucks
 
-1. Change your `nunjucks.configure` property so it only includes the  `node_modules/govuk-frontend/` directory:
+1. In your `nunjucks.configure` property, change `/frontend/` to `/govuk-frontend/`:
```

Comment:
This is no longer correct so I think we need to change this to something like:

When configuring Nunjucks views, remove any references to the `node_modules/govuk-frontend/components` directory.

Reply:
People's configuration file will look like

```js
nunjucks.configure([
 'path/to/their/views',
 'node_modules/govuk-frontend', // Used for the template.njk
 'node_modules/govuk-frontend/components' // Used for components
])
```

And now need to be:

```js
nunjucks.configure([
 'path/to/their/views',
 'node_modules/govuk-frontend' // Used for the everything
])
```